app/models/reservation_scopes.rb

```diff
@@ -50,6 +50,13 @@ def self.included(base) # rubocop:disable MethodLength, AbcSize
       scope :checkoutable, lambda {
         where('start_date <= ?', Time.zone.today).reserved
       }
+      scope :future, lambda {
+        where('start_date > ?', Time.zone.today.to_time).reserved
+      }
+      scope :pending, lambda {
+        where('due_date >= ? and start_date <= ?', Time.zone.today,
```

Comment:
You don't need to query for `due_date >= Time.zone.today` and add the `untouched` scope; you can just use the `reserved` scope instead. In fact, we might just be able to use the `reserved_in_date_range` scope for the controller, so in retrospect we don't need a separate scope (something like '@equipment_model.reservations.reserved_in_date_range(Time.zone.today, Time.zone.today + 8.days).reserved').